Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM 416670754337.dkr.ecr.eu-west-2.amazonaws.com/ci-node-runtime-18
+FROM 416670754337.dkr.ecr.eu-west-2.amazonaws.com/ci-node-build-20
```

Comment:
Should this be using the runtime rather than build image?

Reply:
the value comes from [this PR](https://github.com/companieshouse/dissolution-web/pull/1093/files), which you approved, but happy to change it to a more appropriate value.. 🙂

Reply:
Haha oops, it's not the biggest deal. Just means the image will be a bit bigger than it has to be, I'm happy enough either way but since it was runtime before it suggests we should be able to stick with that

Reply:
will try runtime and see if it build successfully